modules/mgmt-power-remediation-baremetal-about.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+
+// * machine_management/mgmt-power-remediation-baremetal
+
+[id="mgmt-power-remediation-baremetal-about_{context}"]
+= About power-based remediation of bare metal
+In a bare metal cluster, remediation of nodes is critical to ensuring the overall health of the cluster. Physically remediating a cluster can be challenging and any delay in putting the machine into a safe or an operational state increases the time the cluster remains in a degraded state, and the risk that subsequent failures might bring the cluster offline. Power-based remediation helps counter such challenges.
+
+Instead of reprovisioning the nodes, power-based remediation uses a power controller to power off an inoperable node. This type of remediation is also called power fencing.
+
+{product-title} uses the `MachineHealthCheck` controller to detect faulty bare metal nodes by initiating power-cycling. Power-based remediation is fast and reboots unhealthy hosts instead of removing them from the cluster.
```

Comment:
`MachineHealthCheck` controller doesn't detect faulty nodes by initiating power-cycling.

it detect faulty nodes in the same way for all platforms. power-cycling is the remediation action taken in BM.

Reply:
Rephrased to: 
{product-title} uses the `MachineHealthCheck` controller to detect faulty bare metal nodes. Power-based remediation is fast and reboots faulty nodes instead of removing them from the cluster.